lib/tasks/check-links/link_checker.rake

```diff
@@ -1,31 +1,20 @@
 require 'local-links-manager/distributed_lock'
-require 'local-links-manager/check_links/homepage_status_updater'
-require 'local-links-manager/check_links/link_status_updater'
+require 'local-links-manager/check_links/link_status_requester'
 
 desc "Check links"
 task "check-links": :environment do
   service_desc = "Local Links Manager link checker rake task"
-  LocalLinksManager::DistributedLock.new('check-links').lock(
-    lock_obtained: ->() {
-      begin
-        Rails.logger.info("Lock obtained, starting link checker")
-        Services.icinga_check(service_desc, true, "Lock obtained, starting link checker")
+  begin
```

Comment:
That shouldn't be a problem. The Link Checker API automatically caches results for 24 hours for us anyway, and we only need to perform the queries if the last checked time has been updated.